### PR TITLE
Removedjmp

### PR DESCRIPTION
**Pull request by jsvrcek**
removed DJMP as a dependency
Added environment variable for development environment

### CODE REVIEW

File: config/supervisord.conf [hunk 1]
```diff
@@ -5,6 +5,7 @@ file=/var/run/supervisor.sock
 pidfile=/var/run/supervisor.pid
 logfile=/var/log/supervisor.log
 logfile_backups=1
+environment=DEVELOPMENT=True
```
Comment (mvanveen) [inline config/supervisord.conf]: FYI I've got a private repo where I've got a different `supervisord.conf` with the production CF rabbit and postgres credentials inlined into `environment`.  Using it so I can make AWS celery instance provisioning turn-key.  I can invite you to the repo if you want.

File: cf/requirements.txt [hunk 2]
```diff
@@ -24,6 +24,6 @@ whitenoise==2.0.6
 #imposm
 numpy==1.11.1
 -e git://github.com/terranodo/osm-export-tool2.git@cf-tests#egg=oet2
--e git://github.com/terranodo/mapproxy.git@addGeopackageAutoconfig#egg=mapproxy
--e git://github.com/terranodo/django-mapproxy.git@master#egg=djmp
```
Comment (mvanveen) [inline cf/requirements.txt]: 👏  this makes me happy. 😄 

Does this mean that `eventkit-cloud` doesn't need the guardian permissions framework enabled?

File: .gitignore [hunk 3]
```diff
@@ -3,4 +3,5 @@
 .idea
 .vagrant
 ubuntu-xenial-core-cloudimg-amd64-root.tar.gz
-.Dockerfile_eventkit.swp
\ No newline at end of file
+.Dockerfile_eventkit.swp
+manifest.yml
```
Comment (mvanveen) [inline .gitignore]: why is `manifest.yml` in the `.gitignore`?  Don't we currently keep it in the repo?

File: cf/requirements.txt (hunk 2 above)
Comment (jsvrcek) [inline cf/requirements.txt]: we'll add it back when we are ready to use it
.

File: config/supervisord.conf (hunk 1 above)
Comment (jsvrcek) [inline config/supervisord.conf]: meh, no worries, the biggest thing is that users of this repo, can do vagrant or docker-compose up, and be good to go.

**Pull request merged**